cmds/ip/ip.go

```diff
@@ -137,12 +137,13 @@ func showLinkAddresses(w io.Writer, link netlink.Link) {
 		fmt.Fprintf(w, " scope %s %s\n", addrScopes[netlink.Scope(addr.Scope)], addr.Label)
 
 		var validLft, preferredLft string
-		if addr.PreferedLft == math.MaxUint32 {
+		// TODO: fix vishnavanda/netlink. *Lft should be uint32, not int.
```

Comment:
Did you open a bug with them?

Reply:
oh woops misunderstood. Yeah I'll do that.